pkgs/applications/science/programming/scyther/default.nix

```diff
@@ -0,0 +1,86 @@
+{ stdenv, lib, fetchFromGitHub, glibc, flex, bison, python27Packages, graphviz, cmake
+, includeGUI ? false
+, includeProtocols ? true
+}:
+let
+  version = "v1.1.3";
+in
+stdenv.mkDerivation {
+  name = "scyther-${version}";
+  src = fetchFromGitHub {
+    rev = "v1.1.3";
+    sha256 = "0rb4ha5bnjxnwj4f3hciq7kyj96fhw14hqbwl5kr9cdw8q62mx0h";
+    owner = "cascremers";
+    repo = "scyther";
+  };
+
+  buildInputs = [
+    cmake
+    glibc.static
+    flex
+    bison
+  ] ++ lib.optional includeGUI [
+    python27Packages.wrapPython
+  ];
+
+  patchPhase = ''
+    # Since we're not in a git dir, the normal command this project uses to create this file wouldn't work
+    printf "%s\n" "#define TAGVERSION \"${version}\"" >> src/version.h
+  '' + lib.optionalString includeGUI ''
+    file=gui/Scyther/Scyther.py
+    
+    # By default the scyther binary is looked for in the directory of the python script ($out/gui), but we want to have it look where our cli package is
+    substituteInPlace $file --replace "return getMyDir()" "return \"$out/bin\""
+
+    # Removes the Shebang from the file, as this would be wrapped wrongly
+    sed -i -e "1d" $file
+  '';
+
+
+  configurePhase = ''
+    pushd src
```

Comment:
`(cd src && cmakeConfigurePhase) ` (including the parentheses does the same, right? (And is shorter and more portable.)